app/controllers/provider/admin/dashboard/potential_upgrades_controller.rb

```diff
@@ -24,8 +24,8 @@ def current_upgrades
   def is_set_up_correctly?
     usage_limits = current_account
       .application_plans.joins(:usage_limits)
-      .grouping{ issuer_id }.unscope(:order)
-      .references(:usage_limits).count{ usage_limits.id }
+      .grouping { issuer_id }.unscope(:order)
+      .references(:usage_limits).select { id }.count
```

Comment:
This one is weird, the error is: 
```
Mysql2::Error: Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column '3scale_system_test.plans.id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by: SELECT `plans`.* FROM `plans` INNER JOIN `usage_limits` ON `usage_limits`.`plan_id` = `plans`.`id` AND `usage_limits`.`plan_type` = 'Plan' INNER JOIN `services` ON `plans`.`issuer_id` = `services`.`id` WHERE `plans`.`type` IN ('ApplicationPlan') AND `services`.`account_id` = 258 AND `plans`.`issuer_type` = 'Service' GROUP BY `plans`.`issuer_id`
```

Rails 5.0 generates this query:

```sql
SELECT COUNT(*) AS count_all, `plans`.`issuer_id` AS plans_issuer_id FROM `plans` INNER JOIN `usage_limits` ON `usage_limits`.`plan_id` = `plans`.`id` AND `usage_limits`.`plan_type` = 'Plan' INNER JOIN `services` ON `plans`.`issuer_id` = `services`.`id` WHERE `plans`.`type` IN ('ApplicationPlan') AND `services`.`account_id` = 429 AND `plans`.`issuer_type` = 'Service' AND `plans`.`type` IN ('ApplicationPlan') AND `plans`.`type` IN ('ApplicationPlan') GROUP BY `plans`.`issuer_id`
```

New query generates this:
```sql
SELECT COUNT(`plans`.`id`) AS count_all, `plans`.`issuer_id` AS plans_issuer_id FROM `plans` INNER JOIN `usage_limits` ON `usage_limits`.`plan_id` = `plans`.`id` AND `usage_limits`.`plan_type` = 'Plan' INNER JOIN `services` ON `plans`.`issuer_id` = `services`.`id` WHERE `plans`.`type` IN ('ApplicationPlan') AND `services`.`account_id` = 435 AND `plans`.`issuer_type` = 'Service' GROUP BY `plans`.`issuer_id`
```

They are the same, with the exception of condition ``plans`.`type` IN ('ApplicationPlan')` which is only used once as it should be